extensions/api/observability/README.md

```diff
@@ -0,0 +1,35 @@
+# Connector Observability
+
+### `/check/health` endpoint
+
+returns information about the system status. This is primarily intended to be used
+in [`HEALTHCHECK` instructions in Docker files](https://docs.docker.com/engine/reference/builder/#healthcheck). Here,
+this endpoint always returns HTTP 200 indicating that the system is healthy as soon as this endpoint is reachable.
+
+### `/check/liveness` endpoint
+
+From the [Kubernetes docs](https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#container-probes):
+> indicates whether the container is running.
+
+Basically the same semantics as the `/health` endpoint. We can assume, that in the event that the connector crashes, all
+REST endpoints become unavailable.
+
+### `/check/readiness` endpoint
+
+From the [Kubernetes docs](https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#container-probes):
```

Comment:
consider adding a link to https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/ (perhaps replacing the link provided)

Reply:
The site I linked has a good definition of the three K8S probes, plus it linkes to the link you provided anyways.